internal/controllers/machine/machine_controller.go

```diff
@@ -121,15 +122,14 @@ func (r *Reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, opt
 	c, err := ctrl.NewControllerManagedBy(mgr).
 		For(&clusterv1.Machine{}).
 		WithOptions(options).
-		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(mgr.GetScheme(), predicateLog, r.WatchFilterValue)).
+		WithEventFilter(predicates.ResourceHasFilterLabel(mgr.GetScheme(), predicateLog, r.WatchFilterValue)).
 		Watches(
 			&clusterv1.Cluster{},
 			handler.EnqueueRequestsFromMapFunc(clusterToMachines),
 			builder.WithPredicates(
 				// TODO: should this wait for Cluster.Status.InfrastructureReady similar to Infra Machine resources?
 				predicates.All(mgr.GetScheme(), predicateLog,
 					predicates.Any(mgr.GetScheme(), predicateLog,
-						predicates.ClusterUnpaused(mgr.GetScheme(), predicateLog),
 						predicates.ClusterControlPlaneInitialized(mgr.GetScheme(), predicateLog),
```

Comment:
Let's talk about this one too. The Any previous lead to us ignoring ClusterControlPlaneInitialized most of the time.